january-2020/Merge-System-Update.md

```diff
@@ -0,0 +1,12 @@
+Some of you may be aware back in October 2016 with the release of MyBB 1.8.8 & Merge System 1.8.8 we ended the support for the current MyBB Merge System as it stands as we were hoping to rebuild the system from the ground up with the release of MyBB 2.0. As MyBB 2.0 has been put on hold indefinitely and a growing number of users are experiencing problems using the Merge System in it's current form when trying to switch to MyBB we should resume support for the foreseeable future.
```

Comment:
This is quite a long sentence. May be better worded as:

> Some of you may be aware back in October 2016 with the release of MyBB 1.8.8 & Merge System 1.8.8 we ended the support for the current MyBB Merge System. The plan at that time was to rebuild the system from the ground up alongside the release of MyBB 2.0. As MyBB 2.0 has been put on hold indefinitely and a growing number of users are experiencing problems using the Merge System, were planning to reverse that decision and continue supporting the existing Merge System for the foreseeable future.

Reply:
Definitely. Don't know what I was thinking with the original, thanks Euan. If there's no further comments later tonight/tomorrow I will go ahead and publish this post.